.github/workflows/main.yml

```diff
@@ -21,17 +21,15 @@ jobs:
     steps:
       - name: Clone repo
         uses: actions/checkout@v4
-      - name: Install pnpm
-        uses: pnpm/action-setup@v3
-        with:
-          version: 9
+      - name: Enable Corepack
+        run: 'corepack enable'
```

Comment:
@aquarat  can I ask why corepack instead of the pnpm action? With the pnpm action, the version is optional when `packageManager` is specified in package.json ([docs here](https://github.com/pnpm/action-setup/tree/v4/?tab=readme-ov-file#version)).

Reply:
Hey Derek
I'm only seeing this now. I didn't realise there was a pnpm action - if it's better, feel free to use that instead.